### PR TITLE
deleted some 'any' and committed code

### PR DESCRIPTION
**Pull request by makovii**

### CODE REVIEW

File: src/request/request.service.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ import { LogService } from '../log/log.service';
 import { User } from '../user/user.model';
 import * as Response from '../response.messages';
 import { EventGateway } from '../events/events.gateway';
+import { Request as expressRequest}  from 'express';
```
Comment (pavel-kont) [inline src/request/request.service.ts]: interfaces and types conventionally start with uppercase

File: src/user/user.service.ts [hunk 2]
```diff
@@ -261,7 +264,9 @@ export class UserService {
     }
   }
 
-  async ban(req: any, input: BanDto) {
+  async ban(req: Request, input: BanDto) {
+    const objUser = req.user as User;
```
Comment (pavel-kont) [inline src/user/user.service.ts]: a lot of repeats of that line. Can you perhaps override Request obj here - to extend it with user object

File: request-type.d.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { Request } from 'express'
```
Comment (pavel-kont) [inline request-type.d.ts:1]: what is it?

File: request-type.d.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+import { User } from '/user/user.model'
+
+interface RequestdWithUser extends {user: User} {
```
Comment (pavel-kont) [inline request-type.d.ts]:
```suggestion
interface RequestWithUser extends {user: User} {
```

Comment (pavel-kont) [reply request-type.d.ts]: it extends {user: User} but in it's declaration there's
` user: user; ` . what's that user and why the field is overwritten?

File: src/auth/strategies/google.strategy.ts [hunk 5]
```diff
@@ -21,7 +26,8 @@ export class GoogleStrategy extends PassportStrategy(Strategy, 'google') {
     });
   }
 
-  async validate (accessToken: string, refreshToken: string, profile: any, done: VerifyCallback): Promise<any> {
+  async validate (accessToken: string, _refreshToken: string, profile: Profile, done: VerifyCallback): Promise<void> {
+    console.log(profile);
```
Comment (pavel-kont) [inline src/auth/strategies/google.strategy.ts]: rm log

File: src/events/events.gateway.ts [hunk 6]
```diff
@@ -26,8 +32,13 @@ export class EventGateway {
   @WebSocketServer()
   server: Server;
 
-  async handleConnection(socket: any) {
+  async handleConnection(socket: Socket) {
     const payload = socket.handshake.headers.authorization;
+
+    if (!payload) {
+      throw {}
+    }
```
Comment (pavel-kont) [inline src/events/events.gateway.ts]: throw empty obj?

File: src/pipe/validation.pipe.ts [hunk 7]
```diff
@@ -11,9 +11,9 @@ import * as Response from '../response.messages';
 
 @Injectable()
 export class ValidationPipe implements PipeTransform {
-  async transform(value: any, metadata: ArgumentMetadata): Promise<any> {
+  async transform(value: {[key: string]: string}, metadata: ArgumentMetadata): Promise<{[key: string]: string}> {
```
Comment (pavel-kont) [inline src/pipe/validation.pipe.ts]: {[key: string]: string} -> Record<string, string>

File: src/request/request.service.ts [hunk 8]
```diff
@@ -167,40 +164,45 @@ export class RequestService {
...
-        from: input.from,
-        description: input.description,
-        type: RequestType[RequestType.JOIN],
-        status: RequestStatus[RequestStatus.APPROVE] 
-      });
+      if (input.status === RequestStatus.DECLINE) {
+        return Response.SUCCESS
+      } 
+      else {
+        //console.log('!!!!!!!!!!!!!!!!!!!')
```
Comment (pavel-kont) [inline src/request/request.service.ts]: rm

File: src/request/request.service.ts [hunk 9]
```diff
@@ -167,40 +164,45 @@ export class RequestService {
...
+        const successAdding = await this.userService.addToTeam(team, input.from);
+        if(!successAdding) {
+          return new HttpException(Response.FAILED_CHANGE_REQ, HttpStatus.INTERNAL_SERVER_ERROR);
+        }
+        //console.log(successAdding, '??????????????????')
```
Comment (pavel-kont) [inline src/request/request.service.ts]: rm

File: src/user/user.service.ts [hunk 10]
```diff
@@ -14,7 +14,8 @@ import { Team } from '../team/team.model';
 import { BanDto } from './dto/ban.dto';
 import * as env from 'env-var';
 dotenv.config({path: `.${env.get('NODE_ENV').required().asString()}.env`});
-
+import { RequestdWithUser } from 'request-type';
+import { Response as EpressResponse} from 'express'
```
Comment (pavel-kont) [inline src/user/user.service.ts]:
```suggestion
import { Response as ExpressResponse} from 'express'
```

File: src/user/user.service.ts [hunk 11]
```diff
@@ -181,10 +177,13 @@ export class UserService {
     return Response.SUCCESS
   }
 
-  async getImage(req: any, res: any) {
+  async getImage(req: RequestdWithUser, res: EpressResponse) {
     const user = await this.userRepository.findOne({ where: {id: req.user.id} });
 
-    return await res.sendFile(user?.pathPhoto, { root: './images' })
+    if (!user?.pathPhoto) {
+      throw {}
```
Comment (pavel-kont) [inline src/user/user.service.ts]: throw empty obj again?

File: test/player-team.controller.spec.ts [hunk 12]
```diff
@@ -64,15 +64,9 @@ describe('palyer-team', () => {
...
 
-    socketAdmin.on('connection', function(data) {
-      console.log('connect Admin: ', data);
-    });
-    socketManager.on('connection', function(data) {
-      console.log('connect Manager: ', data);
-    });
-    socketPlayer.on('connection', function(data) {
-      console.log('connect Player: ', data);
-    });
```
Comment (pavel-kont) [inline test/player-team.controller.spec.ts]: you can keep logs here

Comment (pavel-kont) [reply test/player-team.controller.spec.ts]: just type data

File: request-type.d.ts (hunk 3 above)
Comment (makovii) [reply request-type.d.ts:1]: request type from express
I found this in the documentation for nestJS

**Review by pavel-kont — APPROVED** [review]

**Pull request merged**